js/logo.js

```diff
@@ -9324,6 +9324,11 @@ function Logo () {
                     that.blocks.blockList[blk].value = that.notesPlayed[turtle][0] / that.notesPlayed[turtle][1];
                 }
                 break;
+            case 'pitchinhertz':
+                if (that.inStatusMatrix && that.blocks.blockList[that.blocks.blockList[blk].connections[0]].name === 'print') {
+                    that.statusFields.push([blk, 'pitchinhertz']);
+                }
```

Comment:
else {
    that.blocks.blockList[blk].value = ...
}

Reply:
I think it should return the frequency in Hertz of the note being played. (You can only return on value, since it can be a chord, so just return the first entry in the array.)